hazelcast/src/test/java/com/hazelcast/map/impl/nearcache/NearCacheTest.java

```diff
@@ -608,16 +713,7 @@ public void testAfterExecuteOnEntriesNearCacheIsInvalidated() {
         populateNearCache(map, mapSize);
 
         EntryObject e = Predicates.newPredicateBuilder().getEntryObject();
-        Predicate predicate = e.get("salary").equal(0);
-        map.executeOnEntries(
-                entry -> {
-                    Employee employee = entry.getValue();
-                    double currentSalary = employee.getSalary();
-                    double newSalary = currentSalary + 10;
-                    employee.setSalary(newSalary);
-                    return newSalary;
-                }, predicate);
-
+        map.executeOnEntries(new TestObjectBasedReadOnlyProcessor());
```

Comment:
This change is not correct, in this test we expect to execute on entries to invalidate the cache, but since the entry processor is now read-only, it doesn't work.